base/range.jl

```diff
@@ -53,11 +53,16 @@ Given a starting value, construct a range either by length or from `start` to `s
 optionally with a given step (defaults to 1, a [`UnitRange`](@ref)).
 One of `length` or `stop` is required.  If `length`, `stop`, and `step` are all specified, they must agree.
 
-If `length` and `stop` are provided and `step` is not, the step size will be computed
-automatically such that there are `length` linearly spaced elements in the range (a [`LinRange`](@ref)).
+In most cases if `step` and `stop` are provided and `length` is not, the overall range length will be computed automatically such that the elements are `step` spaced (a [`StepRange`](@ref)).
 
-If `step` and `stop` are provided and `length` is not, the overall range length will be computed
-automatically such that the elements are `step` spaced (a [`StepRange`](@ref)).
+In case the elements are not simple numeric, elements are spaced using (a [`LinRange`](@ref))
+# Example
```

Comment:
The last sentence should probably be finished with a period, and also the brackets look strange. Between the main docstring and the `# Example` header you could also introduce an empty line, though I'm not sure that makes a difference in the end.